### PR TITLE
Website improvements

### PR DESCRIPTION
**Pull request by wtrocki**
Added gitter chat and relative links support

### CODE REVIEW

**Comment by wtrocki** [comment]
screen shot 2017-08-31 at 1 40 04 pm

File: website/index.html [hunk 1]
```diff
@@ -106,13 +112,13 @@
   <!-- /container -->
   <script src="//ajax.googleapis.com/ajax/libs/jquery/1.11.2/jquery.min.js"></script>
   <script>
-    window.jQuery || document.write('<script src="/js/vendor/jquery-1.11.2.min.js"><\/script>')
+    window.jQuery || document.write('<script src="js/vendor/jquery-1.11.2.min.js"><\/script>')
```
Comment (paolobueno) [inline website/index.html:115]: Just wanted to point out that this seems to be requiring jquery twice.

Comment (paolobueno) [reply website/index.html:115]: Ah, I guess it's a fallback to the local copy if the CDN is down?

Web is weird :joy:

Comment (wtrocki) [reply website/index.html:115]: This is well known hack to support file protocol (local dev)

**Comment by wtrocki** [comment]
@austincunningham - WDYT?

**Review by austincunningham — APPROVED** [review]
Verified

**Pull request merged**